src/WCSimConstructCylinder.cc

```diff
@@ -2886,6 +2891,9 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructCylinderNoReplica()
 					  checkOverlaps);
 
 
+	// Tell DetectorConstruction about boundary walls
+	AddBoundaryWallCylinderDimensions(kBoundaryWallODOuterTyvek,
+									  WCRadius, WCLength);
```

Comment:
Need to debug `WCLength`
https://github.com/WCSim/WCSim/blob/cacd2c5a08f2503192f74f4ce861338b9047163e/src/WCSimConstructCylinder.cc#L2719

```
  if (!isODConstructed) WCLength = WCIDHeight + 2*(WCBlackSheetThickness + 1*mm + pmt_blacksheet_offset);
  else WCLength = WCIDHeight + 2*(WCODHeightWaterDepth + WCBlackSheetThickness + WCODDeadSpace + WCODTyvekSheetThickness);
```